.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,7 @@
 ### Checklist
 
 - [ ] Test coverage for new or modified code paths
+- [ ] Mutation testing passing for new or modified functions
```

Comment:
can you add a link to the new testing doc here?
in fact, i think would be a good idea to link to he relevant locations of that doc for each testing step